src/ResourceManager/Automation/Commands.Automation/Common/AutomationPSClientDSC.cs

```diff
@@ -1718,4 +1811,52 @@ private string GetNodeConfigurationListFilterString(string configurationName)
 
         #endregion
     }
+
+
+    internal class DeploymentTemplateParameters
+    {
+        [JsonProperty("$schema")]
+        public string Schema { get; set; }
+
+        public string ContentVersion { get; set; }
+
+        public ParametersObj Parameters { get; set; }
+    }
+
+    internal class ParametersObj
+    {
+        public TemplateParameters vmName { get; set; }
```

Comment:
Property names should not start with lower-case letters. C# uses Pascal case for properties.